docs/semgrep-app/findings.md

```diff
@@ -108,7 +108,7 @@ Semgrep App assists in the triage process through the use of **comments** and **
 | -----------  | ------------ |
 | **Open** | Open findings require action, such as rewriting the code for vulnerabilities, or refactoring the code. Findings are open by default. |
 | **Ignored** | Findings that are ignored will not be acted upon. This may be a false positive or deprioritized issue. Findings can be ignored through Semgrep App (see [Managing triage states](#managing-triage-states-bulk-triage)). There are two types of ignored findings: <dl><dt>Ignored - App</dt><dd>The ignore status was set within Semgrep App.</dd><dt>Ignored - Code</dt><dd>The ignore status was set from the code itself. Using a `nosemgrep` annotation within the code falls under this type.</dd></dl> |
-| **Fixed** | Fixed findings are findings that were detected in a previous scan of a particular branch, but no longer trigger a match in the most recent scan of that same branch. The rule that detected the finding and the code that triggered the match must both be active in the most recent scan. |
+| **Fixed** | Fixed findings are findings that were detected in a previous scan of a particular branch, but no longer trigger a match in the most recent scan of that same branch. The rule that detected the finding and the code that triggered the match must both be active in the most recent scan. Findings can be fixed by either fixing the code such that the rule no longer matches it, or editing the rule such that it no longer matches the code. |
```

Comment:
```suggestion
| **Fixed** | Fixed findings were detected in a previous scan of a particular branch, but no longer trigger a match in the most recent scan of that same branch. The Semgrep rule that matched the finding and the code that triggered the match must both be active in the most recent scan. Change the triage status of a finding to **fixed** by: <ul><li>Fixing the code so the rule cannot match it.</li><li>Editing the Semgrep rule so it no longer matches the code.</li></ul> |
```

Reply:
The result will be the following:

<img width="927" alt="Screenshot 2022-09-19 at 15 05 26" src="https://user-images.githubusercontent.com/26669307/191013322-3b4e04e3-f307-4ff5-a69b-3cbc1ad8014c.png">